### PR TITLE
#2697 Unnecessary blank spaces removed

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [2697](https://github.com/dynawo/dynawo/issues/2697)

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Transformers/BaseClasses.mo [hunk 1]
```diff
@@ -250,14 +250,14 @@ equation
...
     tap.value = pre(tap.value) - 1;
     tTapUp = pre(tTapUp);
     tTapDown = time;
     Timeline.logEvent1(TimelineKeys.TapDown);
   //Transition to "MoveUpN" (only possible from "MoveUp1" or "MoveUpN")
-  elsewhen (pre(state) == State.MoveUp1 or  pre(state) == State.MoveUpN)  and time - pre(tTapUp) >= tNext and pre(tap.value) < tapMax then
+  elsewhen (pre(state) == State.MoveUp1 or  pre(state) == State.MoveUpN) and time - pre(tTapUp) >= tNext and pre(tap.value) < tapMax then
     state = State.MoveUpN;
     tap.value = pre(tap.value) + 1;
     tTapUp = time;
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Transformers/BaseClasses.mo:263]: I put the comment here but why did you modify the png ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/Converter.mo [hunk 2]
```diff
@@ -20,7 +20,7 @@ model Converter "Converter model for grid forming and grid following application
 IdcSourcePu     IdcPu |          |iConvPu                           iPccPu
 -------->-------->----|          |-->-----(RFilter,LFilter)---------->--(RTransformer,LTransformer)---(terminal)
               |       |          |                                |
-UdcSourcePu (Cdc)     |  DC/AC   |  uConvPu         uFilterPu (CFilter)                      uPccPu
+UdcSourcePu (Cdc)    |  DC/AC   |  uConvPu         uFilterPu (CFilter)                     uPccPu
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/Converter.mo:23]: Are you sure of this one ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo [hunk 3]
```diff
@@ -229,7 +229,7 @@ partial model BaseTransformerVariableTap_INIT "Base model for initialization of
 
                I1  r                I2
     U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-  (terminal1)                   |      (terminal2)
+  (terminal1)                  |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo:232]: Are you sure ?

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo [hunk 4]
```diff
@@ -259,7 +259,7 @@ partial model BaseGeneratorTransformer_INIT "Base model for initialization of Ge
 
                I1  r                I2
     U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-  (terminal1)                   |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo:262]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo [hunk 5]
```diff
@@ -21,7 +21,7 @@ model GeneratorTransformer "Two winding transformer with a fixed ratio"
 
                I1  r                I2
     U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-  (terminal1)                   |      (terminal2)
+  (terminal1)                  |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo:24]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo [hunk 6]
```diff
@@ -101,7 +101,7 @@ The transformer has the following equivalent circuit and conventions:<div><br></
 <p style=\"margin: 0px;\"><br></p>
 <pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">               I1  r                I2</span></pre>
 <pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">    U1,P1,Q1 --&gt;---oo----R+jX-------&lt;-- U2,P2,Q2</span></pre>
-<pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">  (terminal1)                   |      (terminal2)</span></pre>
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo:104]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo [hunk 7]
```diff
@@ -19,7 +19,7 @@ model TransformerFixedRatio "Two winding transformer with a fixed ratio"
 
                I1  r                I2
     U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-  (terminal1)                   |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo:22]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo [hunk 8]
```diff
@@ -54,7 +54,7 @@ annotation(preferredView = "text",
 <p style=\"margin: 0px;\"><br></p>
 <pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">               I1  r                I2</span></pre>
 <pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">    U1,P1,Q1 --&gt;---oo----R+jX-------&lt;-- U2,P2,Q2</span></pre>
-<pre style=\"margin-top: 0px; margin-bottom: 0px;\"><span style=\"font-family: 'Courier New'; font-size: 12pt;\">  (terminal1)                   |      (terminal2)</span></pre>
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo:57]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTap.mo [hunk 9]
```diff
@@ -19,7 +19,7 @@ model TransformerVariableTap "Transformer with variable tap to be connected to a
 
                I1  r                I2
     U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-  (terminal1)                   |      (terminal2)
+  (terminal1)                  |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTap.mo:22]: same

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTapXtdPu.mo [hunk 10]
```diff
@@ -19,7 +19,7 @@ model TransformerVariableTapXtdPu "Transformer with variable tap to be connected
 
                  I1  r                I2
       U1,P1,Q1 -->---oo----R+jX-------<-- U2,P2,Q2
-    (terminal1)                   |      (terminal2)
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTapXtdPu.mo:22]: same

**Review by cossartque — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Transformers/BaseClasses.mo (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Transformers/BaseClasses.mo:263]: It was a mistake. I have restored them.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/Converter.mo (hunk 2 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Sources/Converter.mo:23]: It is a mistake, the change has been reversed.

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo (hunk 3 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo:232]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo (hunk 4 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/BaseClasses_INIT.mo:262]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo (hunk 5 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo:24]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo (hunk 6 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo:104]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo (hunk 7 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo:22]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo (hunk 8 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerFixedRatio.mo:57]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTap.mo (hunk 9 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTap.mo:22]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTapXtdPu.mo (hunk 10 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/TransformerVariableTapXtdPu.mo:22]: Reversed

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo (hunk 6 above)
Comment (cossartque) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Transformers/GeneratorTransformer.mo:104]: Not reversed ?

**Review by cossartque — COMMENTED** [review]
One comment remaining

**Review by cossartque — APPROVED** [review]

**Pull request merged**